.circleci/config.yml

```diff
@@ -28,6 +28,8 @@ jobs:
             - v1-mix-cache-<< parameters.tag >>-{{ checksum "mix.lock" }}
       - run: mix deps.get
       - run: mix test --exclude requires_ipv6
+      - run: mix archive.build
+      - run: mix archive.install
```

Comment:
This was more specific to `nerves_bootstrap` since it is distributed as an archive. But you don't need it here
```suggestion
```